### PR TITLE
Added Clarion

### PR DESCRIPTION
**Pull request by sophiabonsu**
Hi @tas50 !
This the pull request for issue #3366 Can you let me know if it is acceptable ?

### CODE REVIEW

File: brands/tourism/hotel.json [hunk 1]
```diff
@@ -121,6 +121,17 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|Clarion": {
+    "countryCodes": ["us"],
+    "nomatch": ["tourism/hotel|Comfort Suites", "tourism/hotel|Comfort Inn"],
```
Comment (tas50) [inline brands/tourism/hotel.json]: These shouldn't be necessary here

Comment (sophiabonsu) [reply brands/tourism/hotel.json]: HI ! I'm sorry, do you mean the whole pull request is necessary, or lines 124 through 126?

Comment (bmillemathias) [reply brands/tourism/hotel.json]: @tas50 means the above line is unnecessary and can be removed.
Edit your commit to remove it.

Comment (sophiabonsu) [reply brands/tourism/hotel.json]: @bmillemathias  thank you !

**Comment by Adamant36** [comment]
Seems to be fixed. So I merged it. Thanks @sophiabonsu and welcome 👍

**Comment by Adamant36** [comment]
Although, looking back it needed a wikidata entry created for it. Instead of going with operators. Which wasn't done. That's no fault of @sophiabonsu's though. I should have noticed it before merging. I'll probably fix it when I have some time if know one else does.

**Comment by sophiabonsu** [comment]
Hi @Adamant36 I searched  the wiki data for a Clarion Hotel entry and it kept rerouting me to the [choice hotel brand page](https://m.wikidata.org/wiki/Q1075788). There is another clarion entry on Wikidata that links to a clarion  location in [Prague](https://m.wikidata.org/wiki/Q67890719) , should I make a completely new entry , or use  the Prague wikidata ?

**Comment by Adamant36** [comment]
No worries. Id say make a new entry. You don't want it to be location specific.

**Pull request merged**